pysatMissions/methods/spacecraft.py

```diff
@@ -173,7 +173,7 @@ def project_ecef_vector_onto_sc(inst, x_label, y_label, z_label,
 
     # TODO(#65): add checks for existence of ecef labels in inst
 
-    x, y, z = OMMBV.project_ecef_vector_onto_basis(
+    x, y, z = OMMBV.project_ECEF_vector_onto_basis(
```

Comment:
```suggestion
    x, y, z = OMMBV.vector.project_onto_basis(
```
The `project_ECEF_vector_onto_basis` change was apparently a typo on my part when I was moving and deprecating the call.